src/relay/op/nn/convolution.cc

```diff
@@ -248,10 +248,17 @@ bool Conv2DTransposeRel(const Array<Type>& types,
   }
   // dilation
   Array<IndexExpr> oshape({dshape_nchw[0], channels, 0, 0});
-  oshape.Set(2, (param->strides[0] * (dshape_nchw[2] - 1) + dilated_ksize_y -
-                 2 * param->padding[0] + param->output_padding[0]));
-  oshape.Set(3, (param->strides[1] * (dshape_nchw[3] - 1) + dilated_ksize_x -
-                 2 * param->padding[1] + param->output_padding[1]));
+  if ( param->padding.size() == 2 ) {
```

Comment:
minor formatting comment. I think we do not need spaces after `(` and before `)`

Reply:
I will fix it.